src/benchmarks/micro/corefx/System.Drawing/Perf_Color.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+using static System.Drawing.Color;
+
+namespace System.Drawing.Tests
+{
+    [BenchmarkCategory(Categories.CoreFX)]
+    public class Perf_Color 
+    {
+        private static readonly Color[] AllKnownColors;
+
+        private readonly Color _field = DarkSalmon;
+
+        static Perf_Color()
+        {
+            AllKnownColors = new[]
```

Comment:
the original benchmark was using reflection to read that https://github.com/dotnet/corefx/blob/9940166bff52155600f25f15da13a83c982526d2/src/System.Drawing.Primitives/tests/Performance/Perf_Color.cs#L18-L22

I want to add CoreRT runs soon, so I did remove the reflection. There is also a chance (very small ofc) that in the future we might add new colors to `System.Drawing.Color` which would be simply seen as a regression (the array would grow and the time as well)